react-events/my-app/src/CustomButton.tsx

```diff
@@ -0,0 +1,16 @@
+type Props = {
+  text: string;
+  color: string;
+  onCustomClick: (text: string) => void;
+};
+
+export default function CustomButton(props: Props) {
+  function handleClick() {
```

Comment:
This function is no longer needed. You are receiving the event handler as a prop so you can just use it within the JSX.